config/webpack.config.js

```diff
@@ -514,23 +497,31 @@ module.exports = function (webpackEnv) {
 						// This loader doesn't use a "test" so it will catch all modules
 						// that fall through the other loaders.
 						{
-							loader: require.resolve('file-loader'),
+							// loader: require.resolve('file-loader'),
```

Comment:
Why not remove this rather than comment out since `file-loader` is no longer required for v5.